README.md

```diff
@@ -2,6 +2,7 @@
 
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.virtuslab/sbt-akka-serialization-helper/badge.svg)](https://maven-badges.herokuapp.com/maven-central/cz.jirutka.rsql/rsql-parser)
 
+![logo_ash_horizontal@4x](https://user-images.githubusercontent.com/25779550/135059025-4cfade5b-bfcb-47e8-872f-8a3d78ce0c25.png)
```

Comment:
Is this identifier stable? 🤔 I guess it should be changed into a github.com link (at least once this branch is merged)

Reply:
From my understanding it is. We will find out soon enough.